index.html

```diff
@@ -0,0 +1,526 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>My Florence</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./assets/bootstrap/bootstrap.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Merriweather:300|Poppins:300,600" rel="stylesheet">
+    <link href="./public/stylesheets/style.css" rel="stylesheet">
+  </head>
+
+  <body>
+  <div id="main">
+
+    <!--1) GREEN BACKGROUND START-->
+    <div class="bg-green">
+
+    <!--HEADER-->
+      <div class="content-wrap lrg-rl-padding" id="header">
+        <header>
+          <p class="text -x-small -color-dark -caps -wide-spacing -bold centered">A Viget Exploration</p>
+          <img class="header-img" src="./public/img/title.png" alt="Improving Medical Adherence">
+        </header>
+        <div class="pills -yellow"></div>
+        <div class="pills -multiple"></div>
+        <div class="pills -double"></div>
+        <div class="pills -purple"></div>
+      </div>
+
+
+    <!--INTRO SECTION-->
+      <div class="content-wrap lrg-rl-padding bg-white" id="intro">
```

Comment:
Like above, I'd suggest that this be a `<section>` and then use a `<div>` if necessary as an inner wrapper.